pages/integrations/content/fr/recompose.md

```diff
@@ -1,3 +1,28 @@
 ---
 title: '[`recompose`](https://github.com/acdlite/recompose)'
+contributors:
+  - kentcdodds
+  - igitscor
 ---
+
+`glamorous` crée des composants simples, si vous souhaitez complexifier
```

Comment:
Nitpick: Maybe split the sentence? The comma use seems strange to me here.

`glamorous` crée des composants simples. Si vous souhaitez complexifier…
Or add:
`glamorous` crée des composants simples mais si vous souhaitez complexifier…